app/src/shared/contexts/StreamrClient/index.jsx

```diff
@@ -60,11 +60,12 @@ function useClientProvider({
     const isMountedRef = useIsMountedRef()
     const hasClient = !!client
     const hasLoaded = !!apiKey || !!(!isAuthenticating && (isAuthenticated || authenticationFailed))
+    const needLoadKey = apiKey == null && isAuthenticated && !isAuthenticating
```

Comment:
Not sure what is the purpose of `needLoadKey` but if you rebase this on latest development, key loading has been removed from this component.